scripts/conda_to_pip.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+"""
+Convert the conda environment.yml to the pip requirements.txt,
+or check that they have the same packages (for the CI)
```

Comment:
is this script called anywhere right now? should add docs on how to do this in contributing.rst?

Reply:
@datapythonista if you want to add to contributing.rst at some point would be good (IOW how to run the script), but not necessary for now